src/typed-method-types/apps.ts

```diff
@@ -208,6 +208,35 @@ export type DatastoreQueryResponse<
     items: DatastoreItem<Schema>[];
   };
 
+export type DatastoreCountArgs<
+  Schema extends DatastoreSchema,
+> =
+  & BaseMethodArgs
+  & {
+    /**
+     * @description The name of the datastore
+     */
+    datastore: Schema["name"];
+    expression?: string;
```

Comment:
Will the descriptions of there fields be included in the documentation?
Should we add descriptions here?

Reply:
There are descriptions in the documentation.
Here's the dev API page https://api.dev.slack.com/methods/apps.datastore.count which has short descriptions of the optional fields.
There's a much more in-depth explanation of the expression fields here (which should be re-used for the count explanation as the expression fields work the same way as query): https://api.slack.com/automation/datastores-retrieve#query

Reply:
Let's do as much as we can to provide at least a basic description to specific fields. If there are relevant links for devs to read up on for more info, let's link them to it.

See for example how we do document fields and parameters and link to more information in our Node SDK: https://github.com/slackapi/node-slack-sdk/blob/main/packages/web-api/src/methods.ts#L1229-L1232